### PR TITLE
added temporary index page

### PR DESCRIPTION
**Pull request by brettgaylor**
Added temporary index page and links to starter content

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b5cc89947331f12/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b5cc89947331f12/output.txt

Total script time: 1.21 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/59fb8789f5551db/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/59fb8789f5551db/output.txt

Total script time: 0.27 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/001010011a3542b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/001010011a3542b/output.txt

Total script time: 1.07 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_cleanup** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/98e0547802e0cf3/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/98e0547802e0cf3/output.txt

Total script time: 0.94 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/4b0270621fafb3a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/4b0270621fafb3a/output.txt

Total script time: 1.48 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/48080da3823373b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/48080da3823373b/output.txt

Total script time: 1.72 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

File: public/index.html [hunk 1]
```diff
@@ -1,14 +1,142 @@
-<!DOCTYPE html>
-<html lang="en">
+<!doctype html>
+<!-- ==========================================================================
```
Comment (jbuck) [inline public/index.html:2]: I think you can delete all of the html comments in this file; they look like remnants from the Thimble template

File: templates/basic/meme.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
+{
+    "targets": [{
+        "id": "Target0",
+        "name": "video-container",
+        "element": "video-container"
+    }],
+    "media": [{
+        "id": "Media0",
+        "name": "Media0",
+        "url": ["https://www.youtube.com/watch?v=Qw9oX-kZ_9k&NR=1&feature=fvwp&butteruid=1346876878570", "http://videos-cdn.mozilla.net/serv/webmademovies/makecorn.ogv?butteruid=1346876878571", "http://videos-cdn.mozilla.net/serv/webmademovies/makecorn.webm?butteruid=1346876878572"],
```
Comment (jbuck) [inline templates/basic/meme.json]: Just have the youtube video in here

File: templates/basic/meme.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                "name": "TrackEvent13"
+            }]
+        }, {
+            "name": "Layer3",
+            "id": "3",
+            "trackEvents": []
+        }]
+    }],
+    "name": "meme",
+    "projectID": "5047b6f90a5eec8f650002f6"
```
Comment (jbuck) [inline templates/basic/meme.json]: delete the name and projectID so that Butter doesn't get confused

File: templates/basic/tour.json [hunk 4]
```diff
@@ -0,0 +1,181 @@
+{
+    "targets": [{
+        "id": "Target0",
+        "name": "video-container",
+        "element": "video-container"
+    }],
+    "media": [{
+        "id": "Media0",
+        "name": "Media0",
+        "url": ["http://videos-origin.mozilla.org/serv/webmademovies/neighbourhood2.webm?butteruid=1346248121973"],
```
Comment (jbuck) [inline templates/basic/tour.json]: You should add an h264 version of this movie as well

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/efec3824e30bb1b/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/efec3824e30bb1b/output.txt

Total script time: 0.81 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

File: public/index.html [hunk 5]
```diff
@@ -1,14 +1,110 @@
...
+        text-transform: uppercase;
+      }
+
+
+      .content {
+        margin-left: auto;
+        margin-right: auto;
+      }
+
+      #lesson {
```
Comment (jbuck) [inline public/index.html:29]: There's a few unused CSS classes in here

File: templates/basic/lego.json [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+                    "left": 30,
+                    "top": 30,
+                    "zindex": 999
+                },
+                "track": "Layer1",
+                "name": "TrackEvent18"
+            }]
+        }]
+    }],
+    "name": "Lego2",
```
Comment (jbuck) [inline templates/basic/lego.json]: Nuke the name and projectID from this json template and the rest as well.

File: public/index.html [hunk 7]
```diff
@@ -1,14 +1,110 @@
...
+        left: 5%;
+        padding-right: 50px;
+        padding-top: 20px;
+        background: url('https://thimble.webmaker.org/s/webstructable/dashedline.png') no-repeat top right;
+      }
+    </style>
   </body>
-</html>
+
+
```
Comment (jbuck) [inline public/index.html:62]: Extra whitespace can be cleaned-up too

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/15372b6c155eaf4/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/15372b6c155eaf4/output.txt

Total script time: 0.76 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

File: templates/basic/lego.json [hunk 8]
```diff
@@ -0,0 +1,319 @@
...
+                    "textUnderline": "",
+                    "left": 30,
+                    "top": 30,
+                    "zindex": 999
+                },
+                "track": "Layer1",
+                "name": "TrackEvent18"
+            }]
+        }]
+    }],
```
Comment (jbuck) [inline templates/basic/lego.json]: The trailing comma here is why these saved projects are failing to load. Fix that and they'll work again

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3d49399f55b4b19/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/3d49399f55b4b19/output.txt

Total script time: 1.23 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/3e684ff7eddac9a/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/3e684ff7eddac9a/output.txt

Total script time: 1.42 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 2

Live output at: http://botio.butyr.org:8000/b47b8237c359f9f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b47b8237c359f9f/output.txt

Total script time: 1.68 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @brett-etherworks received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/1400e302421734c/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/1400e302421734c/output.txt

Total script time: 1.05 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

File: public/index.html [hunk 9]
```diff
@@ -1,14 +1,83 @@
...
+          <li><a href="/templates/basic/?savedDataUrl=meme.json">Video meme!</a></li>
+          <li><a href="/templates/basic/?savedDataUrl=soundscape.json">Soundscape</a></li>
+          <li><a href="/templates/basic/?savedDataUrl=lego.json">Lego MADLIB!</a></li>
+        </ol>
+      </section>
+
+    </div>
+
+
+    <style>
```
Comment (humphd) [inline public/index.html:13]: Move this style block inside head vs. body, it's not valid here.

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/768e4e164c0c2be/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/768e4e164c0c2be/output.txt

Total script time: 1.27 mins

Server is running on http://pr918.butyr.org/templates/basic/
Run the Test Suite: http://pr918.butyr.org/test/

**Pull request closed without merging**